constraints.txt

```diff
@@ -0,0 +1,3 @@
+# Constraints for Python Packages
+# -----------------------------------------------
+# Pin Versions / Version Ranges if necessary.
```

Comment:
As this file is empty, it should be removed.

Reply:
I am opposed of removing this file. We should add some constraints so that we could ensure that test works. 

Potential packages to pin, or at least define working ranges are:

* setuptools
* zc.buildout
* Sphinx